src/datadog/azext_datadog/generated/_help.py

```diff
@@ -0,0 +1,343 @@
+    parameters:
+      - name: --datadog-organization-properties
+        long-summary: |
+            Usage: --datadog-organization-properties linking-auth-code=XX linking-client-id=XX enterprise-app-id=XX
+
+            linking-auth-code: The auth code used to linking to an existing datadog organization.
+            linking-client-id: The client_id from an existing in exchange for an auth token to link organization.
+            enterprise-app-id: The Id of the Enterprise App used for Single sign on.
+      - name: --user-info
+        long-summary: |
+            Usage: --user-info name=XX email-address=XX phone-number=XX
+
+            name: Name of the user
+            email-address: Email of the user used by Datadog for contacting them if needed
+            phone-number: Phone number of the user used by Datadog for contacting them if needed
+    examples:
+      - name: Monitors_Create
+        text: |-
+               az datadog monitor create --name "myMonitor" --sku-name "myMonitor" --location "West US 2" \
+--datadog-organization-properties name="myOrg" enterprise-app-id="00000000-0000-0000-0000-000000000000" \
+linking-auth-code="someAuthCode" linking-client-id="00000000-0000-0000-0000-000000000000" subscription="pro" \
+--monitoring-status "Enabled" --user-info name="Alice" email-address="alice@microsoft.com" phone-number="123-456-7890" \
+--sku-name "drawdown_testing_20200904_Monthly" --tags Environment="Dev" --resource-group "myResourceGroup"
+"""
+
+helps['datadog monitor update'] = """
+    type: command
+    short-summary: "Update a monitor resource."
+    examples:
+      - name: Monitors_Update
+        text: |-
+               az datadog monitor update --name "myMonitor" --monitoring-status "Enabled" --tags Environment="Dev" \
+--resource-group "myResourceGroup"
+"""
+
+helps['datadog monitor delete'] = """
+    type: command
+    short-summary: "Delete a monitor resource."
+    examples:
+      - name: Monitors_Delete
+        text: |-
+               az datadog monitor delete --name "myMonitor" --resource-group "myResourceGroup"
+"""
+
+helps['datadog monitor wait'] = """
+    type: command
+    short-summary: Place the CLI in a waiting state until a condition of the datadog monitor is met.
+    examples:
+      - name: Pause executing next line of CLI script until the datadog monitor is successfully created.
+        text: |-
+               az datadog monitor wait --name "myMonitor" --resource-group "myResourceGroup" --created
+      - name: Pause executing next line of CLI script until the datadog monitor is successfully deleted.
+        text: |-
+               az datadog monitor wait --name "myMonitor" --resource-group "myResourceGroup" --deleted
+"""
+
+helps['datadog refresh-set-password'] = """
+    type: group
+    short-summary: datadog refresh-set-password
+"""
+
+helps['datadog refresh-set-password get'] = """
+    type: command
+    short-summary: "Refresh the set password link and return a latest one."
```

Comment:
The command name `datadog refresh-set-password get` is a bit difficult to understand and CLI command name needs to follow `noun [noun] ... verb` format.

Reply:
Updated.